content/ui-patterns/empty-states.mdx

```diff
@@ -15,22 +15,40 @@ The Blankslate is made up of several elements that work together to inform the u
+
 ### Primary Text
 
-Use primary text to explain the purpose of the empty state, help users feel comfortable to engage with the content, or begin a feature flow. Primary text should sound welcoming, human, and convey the intention of the feature.
+Primary text acts as a title for the empty space. Use primary text to explain the purpose of the empty state, help users feel comfortable to engage with the content, begin a feature flow, or concisely summarize an error state.
+
+If the space is empty because this the feature hasn't been used yet, convey the intention of the feature in a way that sounds welcoming and human. For example, if a user is trying to view a list of their repositories but hasn't created any yet.
+
+If the space is empty because the feature is temporarily empty, convey that the feature is empty because of the nature of the feature. For example, if a user is trying to view notificartions but has no notifications yet.
+
+If the space is empty because something went wrong, concisely summarize the problem. For example, if a user us trying to view a list of their repositories, but repositories can't be fetched due to a database outage.
 
 ### Secondary text
 
-This optional text is used to inform the user about the feature in more detail. Secondary text should be brief and non-redundant if possible. From the text, users should be able to understand the general purpose of the feature and how it may help them accomplish a goal.
+This optional text is used to inform the user about the feature or error in more detail. Secondary text should be brief and non-redundant if possible. From the text, users should be able to understand what steps they might take next.
+
+If the space is empty because this the feature hasn't been used yet, the general purpose of the feature and how it may help them accomplish a goal.
+
+If the space is empty because something went wrong, explain what they can do to fix the problem. If there is nothing the user can do to fix the problem, secondary text can provide additional revelevant context or explain what they can do to get help.
 
 ### Primary action
 
-Blankslates can and are encouraged to use one primary action. This button should lead to a feature or component creation flow. Button copy should be kept brief and descriptive. If a button requires further specification, consider adding an Octicon.
+Blankslates can and are encouraged to use one primary link or action. The action copy should be kept brief and descriptive. If it requires further specification, consider adding an Octicon before the copy.
+
+If the space is empty because this the feature hasn't been used yet, this action should lead to a feature or component creation flow.
+
+If the space is empty because something went wrong, the action should lead to a solution, a way to get more information, or a way to get help.
```

Comment:
Yes, this can be a link to GitHub's status site or a link for reaching out to GitHub support.

I don't think we need to limit that link to only existing in error banners.